### DISCUSSION TITLE
可不可以加一个能够对网站进行备注的功能

### DISCUSSION THREAD

**Discussion by users3524** [Planshit/Tai#263] [Ideas]
可不可以加一个能够对网站进行备注的功能，一些网站不是经常去，网址不熟悉，但在哪里很好奇到底是什么东西，想加一个备注功能让我可以清晰的看到他是干什么的，就在现在的网站名称位置加上就可以，在统计里最好能够显示我的备注名，十分感谢

**Comment by users3524** [comment]
以及以及，能不能出一个删除应用的功能，一些系统功能的应用我不想让他们被统计到，但仅仅忽略了之后分类的时候他们还是会出现，莫名看着就很烦(抱歉)，就想着能不能出一个类似黑名单的功能，让这些进程(应用)就算出现了也不会被统计/显示出来，谢谢大佬

**Reply by jingjianqian** [reply]
+1

**Comment by noberumotto** [comment]
收到

**Comment by noberumotto** [comment]
网站备注功能已在1.5.0.6版本中加入
